recipe-server/client/control/components/action_fields/PreferenceExperimentFields.js

```diff
@@ -63,12 +69,33 @@ export default class PreferenceExperimentFields extends ActionFields {
           <option value="integer">Integer</option>
           <option value="string">String</option>
         </ControlField>
+        <ControlField
+          label="Preference Branch Type"
+          name="arguments.preferenceBranchType"
+          component="select"
+        >
+          <option value="default">Default</option>
+          <option value="user">User</option>
+        </ControlField>
+        {preferenceBranchType === 'user' &&
+          <p className="field-warning">
```

Comment:
Since this element isn't dynamic, we could add it as a static property of the class. That would have React "render" it once, and would let us write something like:
```
  static userPrefWarning = (<p className="field-warning">...</p>);

// in render
{ preferenceBranchType === 'user' && PreferenceExperimentFields.userPrefWarning }
```

Minor nit, I just prefer to keep `render` concise when possible.